### DISCUSSION TITLE
Tokenizer class LlamaTokenizer does not exist or is not currently imported

### DISCUSSION THREAD

**Discussion by FarhanAnis005** [h2oai/h2o-llmstudio#359] [Q&A]
hey i am trying to use TheBloke/Llama-2-7b-chat-fp16 instead of EleutherAI/pythia-2.8b-deduped but is showing this ValueError: Tokenizer class LlamaTokenizer does not exist or is not currently imported.  please somebody help

**Comment by pascal-pfeiffer** [comment]
tracked in https://github.com/h2oai/h2o-llmstudio/issues/358 and https://github.com/h2oai/h2o-llmstudio/issues/362
